packages/core/src/libraries/hook/index.ts

```diff
@@ -54,6 +36,53 @@ export const createHookLibrary = (queries: Queries) => {
     hooks: { findAllHooks, findHookById },
   } = queries;
 
+  const sendWebhooks = async <T extends HookEventPayloadWithoutHookId>(hooks: Hook[], payload: T) =>
+    Promise.all(
+      hooks.map(async ({ id, config, signingKey }) => {
+        consoleLog.info(`\tTriggering hook ${id} due to ${payload.event} event`);
+
+        // Required:  Override the hookId in the payload
+        const json: HookEventPayload = { ...payload, hookId: id };
+        const logEntry = new LogEntry(`TriggerHook.${payload.event}`);
+
+        logEntry.append({ hookId: id, hookRequest: { body: json } });
+
+        // Trigger web hook and log response
+        await sendWebhookRequest({
+          hookConfig: config,
+          payload: json,
+          signingKey,
+        })
+          .then(async (response) => {
+            logEntry.append({
+              response: await parseResponse(response),
+            });
+          })
+          .catch(async (error) => {
+            logEntry.append({
+              result: LogResult.Error,
+              response: conditional(
+                error instanceof HTTPError && (await parseResponse(error.response))
+              ),
+              error: conditional(error instanceof Error && String(error)),
```

Comment:
didn't get this check. i was expecting:

```suggestion
              error: error instanceof Error ? error : String(error)
```

Reply:
Use normalizeError